index.html

```diff
@@ -25,21 +25,22 @@
   <div class='page-top container-fluid'>
     <div class='row'>
       <div class="col-12 text-center m-4">
-        <img class='nav-logo' src='./images/header.png?t=1'>
+        <img class='nav-logo' src='./images/header.png'>
```

Comment:
Please put the `?t=1` back. You need to change it to `?t=1`. Otherwise browser caching will show the old image still.